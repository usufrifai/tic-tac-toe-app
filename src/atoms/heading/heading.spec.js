import React from 'react';
import Heading from './index';
import renderer from 'react-test-renderer';


describe('Header', () => {
    it('should render  Heading Component', () => {
        const component = renderer.create(
            <Heading title="tic tac toe"/>
          );
          let tree = component.toJSON();
          expect(tree).toMatchSnapshot();
    });
  });