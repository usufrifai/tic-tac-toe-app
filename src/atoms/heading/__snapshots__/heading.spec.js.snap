// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render  Heading Component 1`] = `
<div
  className="heading"
  style={
    Object {
      "width": undefined,
    }
  }
>
  <h3>
    tic tac toe
  </h3>
</div>
`;
